lib/src/rules/omit_local_variable_types.dart

```diff
@@ -102,9 +103,8 @@ class _Visitor extends SimpleAstVisitor<void> {
       }
       var iterableType = loopParts.iterable.staticType;
       if (iterableType is InterfaceType) {
-        var iterableInterfaces = DartTypeUtilities.getImplementedInterfaces(
-                iterableType)
-            .where((type) =>
+        var iterableInterfaces = iterableType.implementedInterfaces.where(
```

Comment:
I wonder whether this might not be cleaner and even more correct to use `DartType.asInstanceOf`.

Reply:
I was wondering the same thing, haha. I've added a TODO.